perfrunbook/debug_hw_perf.md

```diff
@@ -138,6 +147,16 @@ Backend stalls are caused when the CPU is unable to make forward progress execut
 6. If back-end stalls due to the cache-system and memory system are the problem, the data-set size and layout needs to be optimized.
 7. Proceed to [Section 6](./optimization_recommendation.md) to view optimization recommendations for working with a large data-set causing backend stalls.
 
+### Drill down Vectorization
+
+Vectorization is accomplished either by SVE or NEON instructions.  SVE vectorization will use 256-bit vectors on Graviton 3 processors, but the scalable nature of SVE makes both the code and binary vector-length agnostic.  NEON vectorization is always a 128-bit vector size, and does not have the predicate feature of SVE.
+
+For SVE instructions there are metrics which describe how many SVE instructions had empty, full and partially-filled SVE predicates: `inst-sve-empty-pkc`, `inst-sve-partial-pkc`, and `inst-sve-full-pkc`.  These metrics apply to all SVE instructions (loads, stores, integer, and floating-point operations).  A single SVE instruction can execute multiple (vectorized) floating point operations on the ALU.  These are counted individually with `flop-sve-pkc`.  For instance: a single SVE `FMUL` instruction on 32-bit floats on Graviton 3's 256-bit vector will increment the `flop-sve-pkc` counter by 8 because there are 8 32-bit integers in the vector.  Some instructions, such as `FMA` (Fused Multiply Add) may excute 2 or more floating point operations per item in the vector.  The SVE flops are counted assuming a full SVE predicate.
```

Comment:
Some fixes:
floating point operations on the ALU. --> floating point operations in the ALU.
For instance --> For example
because there are 8 32-bit integers in the vector --> because 8 32-bit elements can be processed in a 256-bit vector.
may excute 2 or more floating point operations per item --> may execute 2 floating point operations per item

Which SVE flops are you referring to in this statement: "The SVE flops are counted assuming a full SVE predicate."